<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1079</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Language Detector Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background: rgba(0, 145, 177, 1);
	background: light gray;
}</string>
   </property>
   <widget class="QPushButton" name="btnProcessSentiment">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>920</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnProcessSentiment{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="FeedLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>661</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#FeedLabel {
	background: white;
border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrediction">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblPrediction{
	border: 1px solid black;
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Predicted Character:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>640</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#txtOutput {
	background: white;
border-radius: 10px;
}</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>600</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSentimentAnalysis">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>170</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblSentimentAnalysis{
	border: 1px solid black;
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>920</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnStop{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Pause Webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>920</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnStart{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Open Sign Language Detection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>120</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Sentiment:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>920</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnClear{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOutputTemp">
    <property name="geometry">
     <rect>
      <x>1850</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCheckDup">
    <property name="geometry">
     <rect>
      <x>1850</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FeedLabel_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>250</y>
      <width>341</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#FeedLabel_2 {
	background: white;
border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>120</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Sentiment Percentage:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercentage">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>170</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblPercentage{
	border: 1px solid black;
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>250</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sentiment Probability:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnFirst{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnJump{
	border: 1px solid #F8F8F8;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Jump to Question No.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>780</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnGo{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpeech">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSpeech{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Start Text to Speech</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblQuestions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>471</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblQuestions {
	background: white;
border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Questions:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>710</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnNext{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>710</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnPrev{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>710</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnLast{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Last</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPageNum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>710</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#FeedLabel {
	background: white;
border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cboPage">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>780</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#cboPage{
	border: 1px solid #F8F8F8;
	background: #F8F8F8;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpeechStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSpeechStop{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>290</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Negative:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progNegative">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Neutral:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progNeutral">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Positive:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progPositive">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#progPositive{
background: red;
}</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1821</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SPED Sentiment Analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>600</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Question Logs:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtLogs">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>640</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#txtLogs {
	background: white;
border-radius: 10px;
}</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExport">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>920</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnExport{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>920</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnReset{
	border: 1px solid #F8F8F8;
	border-radius: 10px;
	background: #F8F8F8;
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <action name="actionCollect_Images">
   <property name="text">
    <string>Collect Images</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
